svelte/src/routes/v1/lib/draw.ts

```diff
@@ -76,8 +75,6 @@ export function draw(config: ConfigType, svgElement: any, graphData: GraphDataTy
 			.attr('class', 'group-button')
 			.style('fill', 'black')
 			.on('click', (_event: any, data: any) => {
-				// console.log('click');
```

Comment:
I might make a helper function that will do console log, by changing something inside .env. like `devMode` or `debuggingMode` for example